function.rb

```diff
@@ -0,0 +1,9 @@
+
+def function(n)
+  if n == 0
```

Comment:
I would suggest using a ternary operator. It makes the if more readable.
```suggestion
   n == 0 ? 1 :n * function(n - 1)
```